tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -84,14 +88,177 @@ class MatchMulSigmoid : public RewritePattern {
   OperationName sigmoid_name_;
 };
 
+static FailureOr<TFOp> CreateContractionWithBiasAddOp(PatternRewriter &rewriter,
```

Comment:
I may suggest that having a ContractionPatternBase to include this CreateContractionWithBiasAddOp and all the contraction related patterns inherit ContractionPatternBase